.github/CHANGELOG.md

```diff
@@ -1,6 +1,74 @@
 # Release 0.19.0 (development release)
 
 <h3>New features since last release</h3>
+* The generic multimode Gaussian gate ``Ggate`` is now available in the ``sf.ops`` module with the backend choice of ``tf``. N mode ``Ggate`` can be parametrized by a real symplectic matrix S (size 2N * 2N) and a diplacement vector d (size N). You can also get the gradients of this Gaussian gate by using the ``tape.gradient`` of TensorFlow. [(#599)](https://github.com/XanaduAI/strawberryfields/pull/599)
+
+Example:
+```python
+import numpy as np
+import tensorflow as tf
+import strawberryfields as sf
+from thewalrus.symplectic import sympmat
+
+num_mode = 2
+cutoff = 10
+S = tf.Variable(sympmat(num_mode),dtype=tf.complex64)
+d = tf.Variable(np.random.random(num_mode),dtype=tf.complex64)
+
+eng = sf.Engine("tf", backend_options={"cutoff_dim": cutoff})
+prog = sf.Program(2)
+
+with prog.context as q:
+    sf.ops.Ggate(S,d) | (q[0],q[1])
+
+state_out = eng.run(prog).state.ket()
+print(state_out)
+```
+ It needs to pay attention that, in order to update the two parameters S and d within a optimization task, you can apply the gradients of d on any optimizer, not with the S. (Because the real symplectic matrix is endowed in a Riemannian mannifold, you need to follow its rule to update.) Therefore, there is a function ``update_symplectic`` is prepared inside the library to update the S.
```

Comment:
"Note that in order to update the parameter S from its gradient, you cannot use gradient descent directly (because it would not be symplectic after the update). Please use the function ``update_symplectic`` which is specifically for this purpose."